src/app/protected/error.jsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+import { signIn } from "next-auth/react";
+
+const protectedError = () => {
+  return (
+    <>
+      <p>401! You are not authorized to view this page!</p>
```

Comment:
this isn't guaranteed to be 401, 500 and 403 errors will also route here, so we need to pull from the component props. we should also remove sign in since on a 500 error signing in won't fix the problem